### ISSUE TITLE
[Website Request] Animetsu

### ISSUE THREAD

**Issue by Loidauk** [MALSync/MALSync#2565]
### Website name

Animetsu

### Website link

https://animetsu.cc/

### Website language(s)

English, Romaji

**Comment by crnobog69** [comment]
That would be nice

**Comment by henrik9999** [comment]
It's already supported this issue was just forgotten i think

**Comment by Wkrotik** [comment]
> It's already supported this issue was just forgotten i think

Yeah I noticed the other already closed issue, why is it under the name of gojo though?

**Comment by henrik9999** [comment]
Gojo is what it was called before
